content/references/rippled-api/transaction-formats/transaction-types/signerlistset.md

```diff
@@ -51,6 +51,8 @@ You can create, update, or remove a SignerList using the master key, regular key
 
 You cannot remove the last method of signing transactions from an account. If an account's master key is disabled (it has the [`lsfDisableMaster` flag](accountroot.html#accountroot-flags) enabled) and the account does not have a [Regular Key](cryptographic-keys.html) configured, then you cannot delete the SignerList from the account. Instead, the transaction fails with the error [`tecNO_ALTERNATIVE_KEY`](tec-codes.html).
 
+With the [MultiSignReserve Amendment](known-amendments.html#multisignreserve) enabled, creating a SignerList, or replacing or removing a SignerList that was created before the MultiSignReserve amendment was enabled, sets the lsfOneOwnerCount flag on the SignerList object. When this flag is set, the XRP Ledger is able to lower the [`OwnerCount`](accountroot.html#accountroot-fields) and [owner reserve](reserves.html#owner-reserves) for a SignerList as provided by the MultiSignReserve amendment. For more information, see [SignerList Flag](signerlist.html#signerlist-flag).
```

Comment:
Nit: I try to use "enables a flag" rather than "sets a flag" to avoid confusion.